etna/core/models/mixins.py

```diff
@@ -199,6 +202,52 @@ class Meta:
     base_form_class = RequiredHeroImagePageForm
 
 
+class AccentColourMixin(models.Model):
+    """Mixin to add accent_colour attribute to a Page."""
+
+    accent_colour = models.CharField(
+        max_length=20,
+        default=BrandColourChoices.NONE,
+        verbose_name=_("page accent colour"),
+        help_text=_("The accent colour of the page where relevant."),
+        choices=BrandColourChoices.choices,
+    )
+
+    class Meta:
+        abstract = True
+
+    content_panels = [
+        FieldPanel("accent_colour"),
```

Comment:
Could we maybe move this to another tab, maybe called "Design" as it is a field that affects the page as a whole?